src/main/java/com/nopcommerce/demo/pages/ApparelPage.java

```diff
@@ -61,4 +69,19 @@ public void setVerifyTextClothingText()
         Assert.assertEquals(actualMessage,expectedMessage);
     }
 
+    public void setAccessoriesLink()
+    {
+        log.info("MouseHover On Accessories " + accessoriesLink.toString());
+        mouseHoverToElementAndClick(accessoriesLink);
+    }
+
+    public void setVerifyTextAccessories()
+    {
+        String expectedMessage = "Accessories";
+        String actualMessage = getTextFromElement(verifyTextAccessories);
+        Assert.assertEquals(actualMessage,expectedMessage);
+        log.info("Verify text Accessories " + verifyTextAccessories.toString());
```

Comment:
Can you change text to Text

Reply:
which "text"  need to  change it i didn't get it.

Reply:
I resolved it. Just check for me
Thank you